telegram/request/_httpxrequest.py

```diff
@@ -115,8 +118,8 @@ def __init__(
             max_keepalive_connections=connection_pool_size,
         )
 
-        if http_version not in ("1.1", "2"):
-            raise ValueError("`http_version` must be either '1.1' or '2'.")
+        if http_version not in ("1.1", "2", "2.0"):
```

Comment:
can't believe I didn't add this, must be why I went with a `float`|`int` implementation at first

Reply:
May I understand this as "LGTM", then? 😄